projects/power_distribution/inc/pd_fan_ctrl_defs.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "adt7476a_fan_controller.h"
+
+// I2C info - TODO: Figure out what's needed
+#define ADT_I2C_READ_ADDRESS 0x01   // These values need to be set
+#define ADT_I2C_WRITE_ADDRESS 0x02  // These values need to be set
```

Comment:
Also, you might consider passing the I2C address in in the settings instead of hardcoding it (though of course then it'd be hardcoded in main.c or somewhere else instead of this file).